tests/end2end/test_invocations.py

```diff
@@ -417,3 +417,23 @@ def test_referrer(quteproc_new, server, server2, request, value, expected):
             expected = expected.replace(key, str(val))
 
     assert headers.get('Referer') == expected
+
+
+@pytest.mark.qtwebkit_skip
+@utils.qt514
+def test_preferred_colorscheme(request, quteproc_new):
+    """Make sure the the preferred colorscheme is set."""
+    args = _base_args(request.config) + [
+        '--temp-basedir',
+        '-s', 'colors.webpage.prefers_color_scheme_dark', 'true',
+    ]
+    quteproc_new.start(args)
+
+    quteproc_new.open_path("qute://version")
```

Comment:
`_base_args` above already opens `about:blank`, so I don't think this is needed.

Reply:
You're right. For some reason I thought that `about:blank` had no JS context. Woops.